src/sparseml/modifiers/obcq/base.py

```diff
@@ -58,8 +60,9 @@ class SparseGPTModifier(Modifier):
     quantize: Union[bool, Dict]
     dampening_frac: Optional[float] = 0.01
     sequential_update: Optional[bool] = True
-    prunen: Optional[int] = 0
-    prunem: Optional[int] = 0
+    mask_structure: str = "0:0"
+    prunen_: Optional[int] = None
+    prunem_: Optional[int] = None
```

Comment:
underscore at the end?

Reply:
This is the expected way to define private variables in the new framework; It's actually a very well known practice to put an underscore at the end; also Pydantic does not like underscores in the front